working/0546-patterns/patterns-feature-specification.md

```diff
@@ -1197,7 +1317,7 @@ switch case, and if-case statements, there is no downwards inference from
 pattern to value and the first step is skipped. Instead, the type of the matched
 value is inferred with no downwards context type and we jump straight to
 inferring the types of the case patterns from that context type. *The intent of
-a matcher pattern is to query the type of the matched value, so it would be
+a refutable pattern is to query the type of the matched value, so it would be
```

Comment:
Yeah, that's a good point. I rewrote this. We still don't provide a downwards inference context for switches, but that's because there are multiple cases and it isn't clear how we would merge those into a single context type schema. But we do provide a downwards inference context for both pattern variable declarations and pattern-if statements since in the latter there is only one pattern.